samples/server/index.js

```diff
@@ -0,0 +1,21 @@
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This is copied over from https://expressjs.com/en/starter/hello-world.html
+
+const express = require('express')
+const app = express()
+const port = 3000
+
+const { test } = require('./metrics')
+
+app.get('/', (_, res) => {
+  res.send('Hey, everytime you reload this page it will record a Glean event! On the server!')
```

Comment:
When will events be sent?

Reply:
Same thing as in the browser, first event recorded is sent immediatelly, after that they will be sent every minute or everytime 10 events are collected. By "first event" here I mean when you start the server, the first client that opens the page records an event and that event gets sent, and then every ten events throughout clients. Overall 100% unfit scheduling for scenario, hehe.

Reply:
We should think about this. The lower events count makes sense on a website, since it's an harsher environemnt and the page might be closed very quickly/on shorter sessions.

A server is a different story: it's usually long running, right? Maybe the standard 500 is a fair amount of events in this case.

Reply:
Yeap, I think the main learning here is: we need to abstract that away and have different defaults for each environment (maybe allowing users to define their own custom scheduling when necessary? arguable). I am alright with this being suboptimal in the sample in deferring that discussing for _the real_ Glean.js. Do you agree?

Reply:
Yes, the sample is fine as it is, this is just a thing to account for in the final design.